Netkan/Sources/Github/GithubReleaseAsset.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using CKAN.Versioning;
+
+namespace CKAN.NetKAN.Sources.Github
+{
+    public sealed class GithubReleaseAsset
+    {
+        public String        Name         { get; }
+        public ModuleVersion Version      { get; set; }
+        public Uri           Download     { get; }
+        public DateTime?     AssetUpdated { get; }
```

Comment:
Maybe time to rename this to `Updated`, since it's now defined as part of an asset?